addons/web_editor/static/src/scss/web_editor.variables.scss

```diff
@@ -479,6 +479,22 @@ $o-color-extras-nesting-selector: '&, .o_colored_level &';
     @return $-palette;
 }
 
+$o_shape_connection_height_short: 2rem !default;
+$o_shape_connection_height_regular: 4rem !default;
+$o_shape_connection_height_tall: 15rem !default;
+
+.o_we_shape {
+    --ConnectionShape--height-short: #{$o_shape_connection_height_short};
```

Comment:
Better to let the variable start with `Shape`, followed by `Connection` (see https://github.com/odoo/odoo/pull/185342#discussion_r1846287233).
This way if we'll have to introduce variables for other shapes types, we could follow the same naming structure, eg:
```scss
--ShapeConnection-[x] 
--ShapeOrigins-[x] 
--ShapeWhatever-[x] 
```

Reply:
My bad, I missed that part. 🙈  I made the change 👍